projects/ion/src/lib/dropdown/dropdown.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="container-options" data-testid="ion-dropdown">
+  <div
+    [ngClass]="{
+      'dropdown-disabled': option.disabled,
+      'dropdown-disabled-selected': option.disabled && option.selected,
+      'dropdown-item-selected': option.selected && !option.disabled,
+      'dropdown-item': !option.selected && !option.disabled
+    }"
+    *ngFor="let option of options; let index = index"
+    (click)="select(option)"
+    (mouseenter)="mouseEnter(option)"
+    (mouseleave)="mouseLeave(option)"
+    [id]="'option-' + index"
+  >
+    <div class="container">
+      <span>{{ option.label }}</span>
+      <ion-icon
+        [type]="option.hovered ? 'close' : 'check'"
```

Comment:
When the dropdown is set to single mode and an option is selected, when hovering over that option, the close icon will appear, giving the impression that the option can be removed by clicking. One suggestion is to remove the icon displayed in single selection mode.

```suggestion
        [type]="option.hovered ? multiple ? 'close': '' : 'check'"
```

Reply:
I need check this with design team.